src/Generation/FlysystemProvidedMimeTypeProvider.php

```diff
@@ -185,6 +185,9 @@ public function provideMimeTypes(): array
             new MimeTypeForExtension('application/vnd.oasis.opendocument.text-template', 'ott'),
             new MimeTypeForExtension('application/STEP', 'step'),
             new MimeTypeForExtension('application/STEP', 'stp'),
+            new MimeTypeForExtension('application/octet-stream', 'dst'),
```

Comment:
I've seen `application/tajima` used for this one

Reply:
@colinodell Thanks for the tip, I'll look into it.

Reply:
@colinodell Okay, I have adjusted the application type. For dst files it is now application/tajima.

Thanks

Reply:
Is there anywhere I can validate or verify that `.dst` should be mapped to `application/tajima`? I tried googling for it, but it came out as non-conclusive. Tajima seems to be the vendor.

Reply:
I will try to find evidence of this. I'll get back to you.

Reply:
Unfortunately, I have not found any clear evidence that a `dst` file uses `application/tajima` in the server context.

My search on [GitHub](https://github.com/search?q=application%2Ftajima&type=code) does find `application/tajima` entries assigned to `dst` files.

In our case, these file extensions really belong to the Tajima, it is about embroidery, but I have not found any software for the Mac where it is indicated that a `dst` file is of type `application/tajima`. 

So I will gladly accept @frankdejonge suggestion.